### PR TITLE
resolvconf: New recipe

### PR DESCRIPTION
**Pull request by krevsbech**

### CODE REVIEW

File: recipes/resolvconf/resolvconf.inc [hunk 1]
```diff
@@ -0,0 +1,67 @@
+## Resolvconf - name server information handler
+## Note that this recipe provide the dhclient-enter-hooks script in the dhclient-hooks package.
+## This is slightly hackish as this will override the dhclient-enter-hooks on target
+## However the dhclient currently used is not reading a hook folder but sigle file.
+## The solution for now if multiple recipes provide the script is to not depend on  dhclient-hooks package
+## and then maually merge the dhclient-enter-hooks script.
```
Comment (kimrhh) [inline recipes/resolvconf/resolvconf.inc]: spelling: maually -> manually

File: recipes/resolvconf/resolvconf.inc [hunk 2]
```diff
@@ -0,0 +1,67 @@
+## Resolvconf - name server information handler
+## Note that this recipe provide the dhclient-enter-hooks script in the dhclient-hooks package.
+## This is slightly hackish as this will override the dhclient-enter-hooks on target
+## However the dhclient currently used is not reading a hook folder but sigle file.
```
Comment (kimrhh) [inline recipes/resolvconf/resolvconf.inc]: what do you mean by "the dhclient currently used" ? spelling: sigle -> a single

File: recipes/resolvconf/resolvconf.inc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+   install -m 0644 README ${D}${docdir}/${P}/
+   install -m 0644 ${S}/man/resolvconf.8 ${D}${mandir}/man8/
+}
+
+PACKAGES =+ "${PN}-config ${PN}-dhclient-hooks"
+
+FILES_${PN}-config = "${sysconfdir}/${PN}/interface-order"
+RDEPENDS_${PN} =+ "${PN}-config ${PN}-dhclient-hooks"
+
+# The packet does not provide a daemon as such, however it must be started before the network
```
Comment (kimrhh) [inline recipes/resolvconf/resolvconf.inc]: spelling: packet -> package

File: recipes/resolvconf/resolvconf.inc (hunk 2 above)
Comment (krevsbech) [inline recipes/resolvconf/resolvconf.inc]: Fixed

File: recipes/resolvconf/resolvconf.inc (hunk 1 above)
Comment (krevsbech) [inline recipes/resolvconf/resolvconf.inc]: Fixed

**Pull request merged**